### PR TITLE
TMA-1515: run the bricks as user apache

### PR DESCRIPTION
**Pull request by jimirocks**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1556/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/201/ : FAILURE in 9m 45s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1557/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/202/ : FAILURE in 9m 18s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1559/ : SUCCESS in 32s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/203/ : FAILURE in 15m 36s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1560/ : SUCCESS in 11s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/204/ : FAILURE in 9m 21s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1561/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/205/ : FAILURE in 9m 26s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1562/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/206/ : SUCCESS in 22m 11s

File: Dockerfile [hunk 1]
```diff
@@ -18,19 +19,33 @@ RUN yum install -y curl which patch make git \
...
 
-# login shell is required by rvm
-RUN /bin/bash -l -c ". /home/updater/.rvm/scripts/rvm && rvm install jruby-9.2.5.0 && gem update --system \
-    && gem install bundler rake"
+RUN groupadd -g 48 apache \
+    && useradd -u 48 -m --no-log-init -r -g apache -G rvm apache
+
+RUN chown apache: /src
+
+RUN rvm install jruby-9.2.5.0 && gem update --system \
```
Comment (bkompis) [inline Dockerfile]: it might be good to pull the ruby version out to a variable

File: Dockerfile [hunk 2]
```diff
@@ -18,19 +19,33 @@ RUN yum install -y curl which patch make git \
...
+    && gem install bundler rake
+
+USER 48
```
Comment (bkompis) [inline Dockerfile]: you can use the user name here as well

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1563/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/207/ : SUCCESS in 22m 07s

File: Dockerfile [hunk 3]
```diff
@@ -2,6 +2,7 @@ FROM harbor.intgdc.com/tools/gdc-java-8-jre:d0f5e08
 
 ARG GIT_COMMIT=unspecified
```
Comment (michalvanco) [inline Dockerfile]: move args and labels at the end of the dockerfile

File: Dockerfile [hunk 4]
```diff
@@ -18,19 +19,33 @@ RUN yum install -y curl which patch make git \
 # Switch to directory with sources
```
Comment (michalvanco) [inline Dockerfile]: move this under section where the user is created... so that you don't have to `chown`

File: Dockerfile (hunk 1 above)
Comment (jimirocks) [reply Dockerfile]: done

File: Dockerfile (hunk 2 above)
Comment (jimirocks) [reply Dockerfile]: done

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: Dockerfile (hunk 3 above)
Comment (jimirocks) [reply Dockerfile]: done

File: Dockerfile (hunk 4 above)
Comment (jimirocks) [reply Dockerfile]: done

**Comment by michalvanco** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1565/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/209/ : FAILURE in 8m 08s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1566/ : SUCCESS in 7s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/210/ : FAILURE in 13m 39s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1567/ : SUCCESS in 7s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/211/ : SUCCESS in 21m 42s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-promote-pipeline/109/ : FAILURE in 8m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/330/ : SUCCESS in 46s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/44/ : SUCCESS in 47s (non-voting)

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-promote-pipeline/110/ : FAILURE in 13m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/331/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/45/ : SUCCESS in 7s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-promote-pipeline/111/ : SUCCESS in 29m 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/332/ : SUCCESS in 30s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/46/ : SUCCESS in 7s (non-voting)

**Pull request merged**